README.md

```diff
@@ -1 +1,283 @@
+
+In addition to refactoring the existing handlers, you will add several new ones related to aggregation and reporting. Throughout this project, you will learn to perform basic SQL operations and explore aggregation tools provided by PostgreSQL.
+
+## Context
+
+Systems become more complex and modernized over time, and the hot-coffee project is no exception. While using a JSON-based database is convenient, it is not scalable and makes maintenance difficult for other developers. Moving to a PostgreSQL database is the better solution.
+
+Fortunately, since the project was initially built using a layered architecture, the transition will be easier. You will need to add a new Data Access Layer (Repositories) and integrate it into the project.
+
+As part of this project, you must design tables correctly and define relationships between them appropriately.
+
+## General Criteria
+
+- Your code MUST be written in accordance with [gofumpt](https://github.com/mvdan/gofumpt). If not, you will be graded `0` automatically.
+- Your program MUST be able to compile successfully.
+- Your program MUST not exit unexpectedly (any panics: `nil-pointer dereference`, `index out of range` etc.). If so, you will get `0` during the defence.
+- Only built-in packages are allowed, except for the PostgreSQL driver. Using any other external packages will result in a grade of 0.
+- The project MUST be run by the following command in the project's root directory:
+
+```shell
+$ docker compose up
+```
+
+## Mandatory Part
+
+Your task is to rewrite the existing endpoints in the **hot-coffee** project to work with a PostgreSQL database. To do this, you may use a third-party PostgreSQL driver as part of your solution.
+
+You need to create the following tables: `user`, `order`, `menu`, `menu_item`, `inventory`, and `ingredient`, and define the relationships between them. You are not limited to only these tables; you may create additional ones if necessary for implementing the required functionality. The design and relationships of the tables are at your discretion.
+
+Pay special attention to designing the relationships between tables (e.g., one-to-many, one-to-one). For instance, the `menu` and `menu_item` tables will have a "one-to-many" relationship, meaning the `menu_item` table should include a column with the following constraint:
+```sql
+menu_id INT NOT NULL CONSTRAINT fk_menu FOREIGN KEY(menu_id) REFERENCES menu(id)
+```
+Plan your data structure carefully to ensure the project's scalability and maintainability.
+
+Since adding a database dependency to your project makes it more challenging for auditor to run and test it, you need to containerize your service and database into separate containers and write a `docker-compose.yml` file. This will allow the project to be started with a single command:
```

Comment:
I think it is too hard for them to containerization for their own, considering it is their first project and that they will learn this topic much later in the course. I think they would spend much more time than intended in this section (I personally would, if it was the first time for me facing this topic)

Either one of this solution could be applied:
1. Provide detailed guide on how to setup the project in docker-compose
2. Do not include docker and leave the testing part as it is, considering that authors would provide mock data in init.sql that would used to quickly setup the database for testing